lib/ansible/modules/network/eos/eos_config.py

```diff
@@ -44,6 +44,8 @@
         in the device running-config.  Be sure to note the configuration
         command syntax as some commands are automatically modified by the
         device config parser.
+      - Abbreviated commands in lines are NOT idempotent, see
```

Comment:
Do we need the same on `parents:`?

Reply:
Should be true for configuration in `src` option.

Reply:
Thanks for the feedback. We could repeat that comment and link in any section where it applies, and/or put it in the Notes section instead of in the parameter descriptions. I'm leaning toward making the information a Note - thoughts?

Reply:
Yes agreed a line in notes section makes more sense instead of adding it in each applicable option